homeassistant/components/synology_dsm/translations/en.json

```diff
@@ -31,14 +31,6 @@
                 "description": "Do you want to setup {name} ({host})?",
                 "title": "Synology DSM"
             },
-            "reauth": {
```

Comment:
I don't think this will have any effect since translations are downloaded from Lokalise.

Reply:
Aren't we suppose to run "`python3 -m script.translations develop`" when strings.json change ?

Simone

Reply:
That's just to ease local development testing. It doesn't affect Lokalise, which is the source of translations.

Reply:
Should we commit anyway (as it doesn't hurt) or close this PR and that's it ?

Reply:
I suggest closing. The infrastructure will sync the translations from Lokalise so it'll probably add this back again until Lokalise is updated.

Reply:
Remove reauth key in Localise instead.